partials/content.php

```diff
@@ -35,7 +35,7 @@
 		<?php the_excerpt(); ?>
 	</div>
 	<?php else : ?>
-	<div class="entry-content" itemprop="articleBody">
+	<div class="entry__content entry__a--transition" itemprop="articleBody">
```

Comment:
removing entry__a--transition. We already decided to go for transition for all. Makes sense? @mitogh